### ISSUE TITLE
Does CodeXL support debugging "-cl-std=2.0" ?

### ISSUE THREAD

**Issue by dragontamer** [GPUOpen-Tools/CodeXL#166]
I've got a simple program that I'm unable to debug through CodeXL. It seems like adding the "-cl-std=2.0" flag (to enable OpenCL 2.0 features) prevents me from being able to debug OpenCL kernels through CodeXL.

If OpenCL 2.0 isn't supported, some documentation would be nice.

* Platform: Windows 10
* Radeon Software: 15.8
* CodeXL: 2.5.67.0
* Visual Studio 2015 Community Edition

The "-cl-std=2.0" flag was all I needed to change in my tests to get debugging to fail. If this doesn't work for you, I can put together a simple project file and upload my test case.

**Comment by dragontamer** [comment]
Ah, I have found the message, although its possibly because I messed up here that there's a weird edge case.

The "proper" flag is "std=CL2.0". And when this is written, there's an error message that states "Debugging is not supported for OpenCL 2.0". That's fair.

Can the "Debugging is not supported for OpenCL 2.0" message come up when the erroneous "-cl-std=2.0" flag is specified? It seems like the OpenCL program is still built correctly as a 2.0 file, the message just isn't carried over for some reason.

![image](https://user-images.githubusercontent.com/3312438/32412727-b788832c-c1ce-11e7-95f6-ee7277c8fa63.png)
